nx/lib/nx/defn/kernel.ex

```diff
@@ -403,6 +403,27 @@ defmodule Nx.Defn.Kernel do
   def left / right when Kernel.and(is_number(left), is_number(right)), do: Kernel./(left, right)
   def left / right, do: Nx.divide(left, right)
 
+
+  @doc """
+  Element-wise remainder operation.
+
+  It delegates to `Nx.remainder/2` (supports broadcasting).
+
+  ## Examples
+
+      defn divides_by_5?(a) do
+        rem(a, 5)
+        |> Nx.any?
+        |> Nx.equal(Nx.tensor(1))
+      end
+
```

Comment:
```suggestion
```

Reply:
@josevalim I just noticed that the other docs in this file tend to end on a blank line. Does this make any difference?

Reply:
No, it does not matter, but that's the "convention" we follow in Elixir (and apparently Nx).

Reply:
I see!